lesson02/elb_asg/common_vars.tf

```diff
@@ -0,0 +1,36 @@
+variable "aws-region" {
+  default = "us-east-1"
+}
+
+variable "metadata" {
+    default = {
+    appname = "sample-app"
+    appversion = "latest"
+  }  
+}
+
+
+variable "env" {
+   description = "string for the enviroment, allowed values develop, uat, prod"
+   default= "develop"
+}
+
+
+variable "vpc_id" {
```

Comment:
This variable is not being used, there is the `data "aws_vpc" "selected"` instead.

Reply:
Same for variables: metrics_granularity, cooldown, iam_role, image_id (some of them are used on the commented code)

Reply:
I want to leave some commented code since its optional do you guys want to remove them ?